src/main/java/io/stargate/sgv2/jsonapi/exception/playing/DatabaseException.java

```diff
@@ -0,0 +1,25 @@
+package io.stargate.sgv2.jsonapi.exception.playing;
+
+public class DatabaseException extends ServerException {
+
+  public static final Scope SCOPE = Scope.DATABASE;
```

Comment:
Same as with other constants:

1. Should not be `public`?
2. Name is very generic/vague, probably `DEFAULT_SCOPE`.
3. ... but is this constant actually needed at all? Seems like its only used from sub-class.

Not 100% sure what to do with it, but just seems wrong. (1 and 2 easy enough to change, but eliminating need would be best)

Reply:
idea was for the Code enum to be able to refence a central definition while it is constructing